tdrs-backend/tdpservice/settings/__init__.py

```diff
@@ -1,2 +0,0 @@
-from .local import Local  # noqa
-from .production import Production  # noqa
```

Comment:
The `production` module was deleted and its class moved in to the `cloudgov` module.